Lib/idlelib/configdialog.py

```diff
@@ -1251,7 +1425,7 @@ def CreatePageExtensions(self):
                                       selectmode='browse')
         self.extension_list.bind('<<ListboxSelect>>', self.extension_selected)
         scroll = Scrollbar(frame, command=self.extension_list.yview)
-        self.extension_list.yscrollcommand=scroll.set
+        self.extension_list.yscrollcommand = scroll.set
```

Comment:
Code changes like should have been left for another patch, but they are here now and very unlikely to cause a problem since the extension tab is relatively new and should not be touched by existing patches.